CONTRIBUTING.md

```diff
@@ -1,65 +1,99 @@
 
 <a name="approvals"></a>
-### Approval Process
+### Pull Requests Approval Process
 
 - Once a PR is created, it needs to get the following approvals before it is merged into master.
-  - From at least one of committers for that workstream
+  - From at least one of contributor for that workstream
     - Approvers should be distributed equally among operators and vendors.
   - From WSL (to make sure consensus is reached).
   - WSL can request additional approvals reviews.
     - Recommendation is to use 4 approvals for complex PRs.
-  - 2 business days cool off period will be applied before Final approval.
+  - 2 business days cool off period should be applied before Final approval.
   - Final Approval by TSL (to make sure process is followed)
-- Only One approval is permitted per each organization for a given PR.
-- The selection of which Committers to approve a PR is made by the WSL and should take those factors into consideration:
-  - Committers needs to be actively discussing the PR to be selected for approval.
-  - Committers needs to be active in the workstream.
+- Only One approval will be counted per each organization for a given PR.
+- The selection of which contributor to approve a PR is made by the WSL and should take those factors into consideration:
+  - Contributor needs to be actively discussing the PR to be selected for approval.
+  - Contributor needs to be active in the workstream.
 - PRs will be merged automatically online by the TSL once consensus is reached and all approvals are received. 
-- If WSL is the person who is creating the PR, They need to request an alternate approver, preferably a co-lead or from the committers list.
-
+- If WSL is the person who is creating the PR, They need to request an alternate approver, preferably a co-lead or from the contributors list.
 
 <p align="right"><img src="artefacts//figures/approval_process.png" alt="scope" title="Scope" width="100%"/></p>
 <p align="center"><b>Figure 1:</b> Approval Process</p>
 
+<a name="idle"></a>
+### Idle Pull Requests Policy
+
+Pull Requests will be automatically labled as "**Idle**" when:
+- No Engagement from the author of the PR to address Review Requests or input from the community for more than 15 calender days.
+  - Personal holidays or public holidays will not be counted.
+- One or more identified Reviewers (including WSL) are not providing feedback/resolutions or approving the Pull Request for more than 15 calender days.
+
+Any Pull Requests that are labeled as "Idle" will be discussed during Technical Steering Meeting to take a decision on, such as:
+- Close the PR and label it as "**Returned**".
+- "**Force**" Merging of the PR without having full reviewers approvals due to their inactivity.
+
+<a name="unresolved"></a>
+### Unresolved Pull Requests Policy
+
+Pull Requests will be automatically labled as "**Unresolved**" when no consensus are reached during approval process.
```

Comment:
typo: "... when no consensus are reached .." ---> "... when no consensus is reached  ..."

Reply:
fixed.